docs/experiment/sdks/ruby-sdk.md

```diff
@@ -168,6 +171,157 @@ experiment.fetch_async(user) do |_, variants|
+
+!!!info "Ruby version compatibility"
+
+    The Ruby Server SDK works with Ruby 2.0+.
+
+=== "bundler"
+
+    ```bash
+    gem 'amplitude-experiment'
+    ```
+
+=== "gem"
+
+    ```bash
+    gem install amplitude-experiment --pre
+    ```
+
+!!!tip "Quick Start"
+
+    1. [Initialize the experiment client](#initialize)
+    2. [Start the local evaluation client](#start)
+    3. [Evaluate a user](#evaluate)
+
+    ```ruby
+    require 'amplitude-experiment'
+
+    # (1) Initialize the local evaluation client with a server deployment key.
+    experiment = AmplitudeExperiment.initialize_local('<DEPLOYMENT_KEY>', AmplitudeExperiment::LocalEvaluationConfig.new)
+
+    # (2) Start the local evaluation
+    experiment.start
+
+    # (3) Evaluate a user
+    user = AmplitudeExperiment::User.new(
+      user_id: 'user@company.com',
+      device_id: 'abcdefg',
+      user_properties: {
+        'premium' => true
+      }
+    )
+    variants = expriment.evaluate(user)
+    variant = variants['YOUR-FLAG-KEY']
+    unless variant.nil?
+        if variant.value == 'on'
+            # Flag is on
+        else
+            # Flag is off
+        end
+    end
+    ```
+
+### Initialize
+
+Initializes a [local evaluation](../general/evaluation/local-evaluation.md) client.
+
+!!!warning "Server Deployment Key"
+    You must [initialize](#initialize_1) the local evaluation client with a server [deployment](../general/data-model.md#deployments) key in order to get access to local evaluation flag configs.
+
+```ruby
+AmplitudeExperiment.initialize_local(api_key)
+```
+
+| Parameter | Requirement | Description |
+| --- | --- | --- |
+| `apiKey` | required | The server [deployment key](../general/data-model.md#deployments) which authorizes fetch requests and determines which flags should be evaluated for the user. |
+| `config` | optional | The client [configuration](#configuration) used to customize SDK client behavior. |
+
+!!!tip "Flag Polling Interval"
+    Use the `flag_config_polling_interval_millis` [configuration](#configuration_1) to determine the time flag configs take to update once modified (default 30s).
+
+#### Configuration
+
+The SDK client can be configured on initialization.
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div> | Description | Default Value |
+    | --- | --- | --- |
+    | `server_url` | The host to fetch flag configurations from. | `https://api.lab.amplitude.com` |
+    | `bootstrap` |	Bootstrap the client with a map of flag key to flag configuration | `{}` |
+    | `flag_config_polling_interval_millis` | The interval to poll for updated flag configs after calling [`start`](#start) | `30000` |
+    | `debug` | Set to `true` to enable debug logging. | `false` |
+
+### Start
+
+Start the local evaluation client, pre-fetching local evaluation mode flag configs for [evaluation](#evaluate) and starting the flag config poller at the [configured](#configuration_1) interval.
+
+```ruby
+start
+```
```

Comment:
is this how you call the function on the client? Or is this the function "definition"

Reply:
I think it's the function "definition".  It's followed by how we call on clients.
I copied from python, 
![Screen Shot 2022-08-22 at 10 30 16 PM](https://user-images.githubusercontent.com/13028329/186077575-b4206da8-54dd-46e4-a850-6daf2be4f329.png)
This is how python shows now.